From a718243fe78fe9002b4145a158e02801e907399b Mon Sep 17 00:00:00 2001
From: Florian Schmaus <flo@geekplace.eu>
Date: Sun, 2 Oct 2022 19:47:47 +0200
Subject: [PATCH 1/2] Declare variable D as local in stop_and_clean

This also fixes tests in Gentoo, where src_test() has already declare
a variable D, containing the path of the temporary install
directory (aka. DESTDIR).
--- a/test/test.sh
+++ b/test/test.sh
@@ -576,6 +576,7 @@ stop_and_clean(){
     ## +3 is for "Test tsize option ..." and "Test PCRE mapped download ... "
     ## +2 for diskspace tests:
     local M=$(grep "/tmp/" "$TDIR"/multicast*.log | wc -l)
+    local D
     $INTERACTIVE && D=2
     cat <<EOF
 Expected:
